plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1669,6 +1688,14 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
     )
   }
 
+  /**
+   * Called when new style is loaded.
+   */
+  override fun onStyleChanged(styleDelegate: StyleInterface) {
+    Logger.e("KIRYLDD", "new style: $styleDelegate")
```

Comment:
```suggestion
    Logger.e(TAG, "new style: $styleDelegate")
```

Reply:
or maybe remove it. it's not an `error` level

Reply:
thanks, already removed 😄